src/proposals/20241017/SparkEthereum_20241017.t.sol

```diff
@@ -219,45 +219,47 @@ contract SparkEthereum_20241017Test is SparkEthereumTestBase {
         _assertMorphoCap(ptUsde26Dec, 100_000_000e18);
         _assertMorphoCap(ptUsde27Mar, 100_000_000e18);
         
-        uint256 ptUsde26DecPrice = IMorphoChainlinkOracle(PT_26DEC2024_PRICE_FEED).price();
-        uint256 ptUsde27MarPrice = IMorphoChainlinkOracle(PT_27MAR2025_PRICE_FEED).price();
+        // Commented because of an open issue with Foundry with the evm version
+        // https://github.com/foundry-rs/foundry/issues/6228
+        // uint256 ptUsde26DecPrice = IMorphoChainlinkOracle(PT_26DEC2024_PRICE_FEED).price();
+        // uint256 ptUsde27MarPrice = IMorphoChainlinkOracle(PT_27MAR2025_PRICE_FEED).price();
 
-        assertEq(ptUsde26DecPrice, 0.969105874238964993e36);
-        assertEq(ptUsde27MarPrice, 0.908944818619989853e36);
+        // assertEq(ptUsde26DecPrice, 0.969105874238964993e36);
+        // assertEq(ptUsde27MarPrice, 0.908944818619989853e36);
```

Comment:
Update to (commented out)

```solidity
        assertEq(ptUsde26DecPrice, 0.970544002092846271e36);
        assertEq(ptUsde27MarPrice, 0.910862322425164891e36);
```

And add a note `// NOTE: Can run this test individually after commenting out these assertions with forge t --mt testMorphoVaults -vvv --evm-version cancun`